pkg/operator/controller/controller_dns_configmap_test.go

```diff
@@ -336,3 +337,1449 @@ foo.com:5353 {
+								Address: "9.8.7.6",
+							},
+							{
+								Type:    operatorv1.NetworkResolverType,
+								Address: "6.4.3.2",
+								Port:    53,
+							},
+							{
+								Type:    operatorv1.NetworkResolverType,
+								Address: "2.3.4.5",
+								Port:    5353,
+							},
+							{
+								Type:    operatorv1.NetworkResolverType,
+								Address: "127.0.0.53",
+							},
+						},
+						Policy: operatorv1.RoundRobinForwardingPolicy,
+					},
+					Servers: []operatorv1.Server{
+						{
+							Name:  "foo",
+							Zones: []string{"foo.com"},
+							ForwardPlugin: operatorv1.ForwardPlugin{
+								Upstreams: []string{"1.1.1.1", "2.2.2.2:5353"},
+								Policy:    operatorv1.RoundRobinForwardingPolicy,
+							},
+						},
+						{
+							Name:  "bar",
+							Zones: []string{"bar.com", "example.com"},
+							ForwardPlugin: operatorv1.ForwardPlugin{
+								Upstreams: []string{"3.3.3.3"},
+								Policy:    operatorv1.RandomForwardingPolicy,
+							},
+						},
+						{
+							Name:  "fizz",
+							Zones: []string{"fizz.com"},
+							ForwardPlugin: operatorv1.ForwardPlugin{
+								Upstreams: []string{"5.5.5.5", "6.6.6.6"},
+								Policy:    operatorv1.SequentialForwardingPolicy,
+							},
+						},
+						{
+							Name:  "buzz",
+							Zones: []string{"buzz.com", "example.buzz.com"},
+							ForwardPlugin: operatorv1.ForwardPlugin{
+								Upstreams: []string{"4.4.4.4"},
+							},
+						},
+					},
+				},
+			},
+			expectedCoreFile: `# foo
+foo.com:5353 {
+    prometheus 127.0.0.1:9153
+    forward . 1.1.1.1 2.2.2.2:5353 {
+        policy round_robin
+    }
+    errors
+    log . {
+        class error
+    }
+    bufsize 512
+    cache 900 {
+        denial 9984 30
+    }
+}
+# bar
+bar.com:5353 example.com:5353 {
+    prometheus 127.0.0.1:9153
+    forward . 3.3.3.3 {
+        policy random
+    }
+    errors
+    log . {
+        class error
+    }
+    bufsize 512
+    cache 900 {
+        denial 9984 30
+    }
+}
+# fizz
+fizz.com:5353 {
+    prometheus 127.0.0.1:9153
+    forward . 5.5.5.5 6.6.6.6 {
+        policy sequential
+    }
+    errors
+    log . {
+        class error
+    }
+    bufsize 512
+    cache 900 {
+        denial 9984 30
+    }
+}
+# buzz
+buzz.com:5353 example.buzz.com:5353 {
+    prometheus 127.0.0.1:9153
+    forward . 4.4.4.4 {
+        policy random
+    }
+    errors
+    log . {
+        class error
+    }
+    bufsize 512
+    cache 900 {
+        denial 9984 30
+    }
+}
+.:5353 {
+    bufsize 512
+    errors
+    log . {
+        class error
+    }
+    health {
+        lameduck 20s
+    }
+    ready
+    kubernetes cluster.local in-addr.arpa ip6.arpa {
+        pods insecure
+        fallthrough in-addr.arpa ip6.arpa
+    }
+    prometheus 127.0.0.1:9153
+    forward . 1.2.3.4 9.8.7.6 6.4.3.2:53 2.3.4.5:5353 127.0.0.53 {
+        policy round_robin
+    }
+    cache 900 {
+        denial 9984 30
+    }
+    reload
+}
+`,
+			expectedError: nil,
```

Comment:
Just to save some lines in this already long file: this field can be omitted if no error is expected, it'll be `nil` by default.